Source/WebKit/UIProcess/WebProcessPool.h

```diff
@@ -725,6 +725,9 @@ class WebProcessPool final
     bool m_shouldTakeUIBackgroundAssertion;
     bool m_shouldMakeNextWebProcessLaunchFailForTesting { false };
     bool m_tccPreferenceEnabled { false };
+#if ENABLE(REMOTE_INSPECTOR)
+    bool m_gpuToolsEnabled { false };
```

Comment:
It looks like this is unused?

Reply:
Oh no! I thought I removed this! Will fix.